app/src/main/java/com/bogdan/codeforceswatcher/components/compose/theme/Shape.kt

```diff
@@ -5,7 +5,7 @@ import androidx.compose.material.Shapes
 import androidx.compose.ui.unit.dp
 
 val Shapes = Shapes(
-    small = RoundedCornerShape(4.dp),
-    medium = RoundedCornerShape(4.dp),
+    small = RoundedCornerShape(100),
```

Comment:
Still worth using `.dp` for consistency?

Reply:
It will be incorrect to set a `.dp` value in this case, so we use `%`. It's Ok.